src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -250,8 +252,8 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       const formattedSocDate = moment(details.lastSocDate, 'YYYY-MM-DD').format('MMMM Do, YYYY');
       contents.title = 'Please review your Supplemental Statement of the Case';
       contents.description = (
-        <p>The Veterans Benefits Administration sent you a Supplemental Statement of the Case on {formattedSocDate}
-        because, after completing the remand instructions from the Board, they couldn’t fully grant
+        <p>The Veterans Benefits Administration sent you a Supplemental Statement of the Case on {formattedSocDate} because,
+          after completing the remand instructions from the Board, they couldn’t fully grant
```

Comment:
it looks like there's an extra space here now.

Reply:
Oh, I should mention why this was done: I opened up an appeal and saw the date backed right into the next word without a space, so I just bumped the next word up a line to make sure it was parsed with a space.